app/components/Form/ApplicationForm.tsx

```diff
@@ -176,6 +176,8 @@ const ApplicationForm: React.FC<Props> = ({
 
   const [reviewConfirm, setReviewConfirm] = useState(false);
   const [savingError, setSavingError] = useState(null);
+  const [saveAsDraft, setSaveAsDraft] = useState(true);
```

Comment:
It took me a few passes to understand what this state does.
`save` is an action, so one could expect `saveAsDraft` to be a function, not a boolean.
Should this be `canSaveAsDraft` ? Or, alternatively, setting it to `false` by default and rename it to `savedAsDraft`?
A comment indicating that this is only for the submission page could be useful, as we're saving drafts on edit.